dbms/src/Storages/DeltaMerge/Segment.cpp

```diff
@@ -3170,47 +3100,95 @@ BitmapFilterPtr Segment::buildMVCCBitmapFilterNormal(
     size_t expected_block_size)
 {
     Stopwatch sw_total;
-    ColumnDefines columns_to_read{
-        getExtraHandleColumnDefine(is_common_handle),
-    };
-    sanitizeCheckReadRanges(__FUNCTION__, read_ranges, rowkey_range, log);
```

Comment:
Keep this `sanitizeCheckReadRanges`? Many unittest call `buildMVCCBitmapFilter` directly, which is not covered this check.